src/rp2_common/pico_aon_timer/include/pico/aon_timer.h

```diff
@@ -23,6 +23,23 @@
  * \if rp2350_specific
  * This library uses the Powman Timer on RP2350.
  * \endif
+ *
+ * This library supports both `aon_timer_xxx_calendar()` methods which take a calendar date/time (as struct tm),
+ * and `aon_timer_xxx()` methods which take a time since epoch (as struct timespec)
+ *
+ * \if rp2040_specific
+ * NOTE: On RP2040 the non date/time methods must convert the struct timespec to a date/time value which is handled via
```

Comment:
I don't really understand what it means by "non date/time methods" here

Reply:
maybe a list of the methods that are affected by the clib overhead in 2040 and 2350 (a different set?) would be useful?

Reply:
can you suggest a clarification - this refers very much to the previous paragraph (i could add "calendar" here)

I suppose we could list them all